CHANGELOG.md

```diff
@@ -16,6 +16,8 @@
 
 - Add locales for the ended session, version bump asab-webui (INDIGO Sprint 230512, [!34](https://github.com/TeskaLabs/seacat-admin-webui/pull/34))
 
+- Version bump asab-webui (INDIGO Sprint 230526, [!36](https://github.com/TeskaLabs/seacat-admin-webui/pull/36))
```

Comment:
@Aringoaway Version bumps written separately in the changelog does not have any meaning, since it has no value to the person who is reading the changelog